pages/linux/betterlockscreen.md

```diff
@@ -13,7 +13,7 @@
 
 - Lock the screen, showing some custom text:
 
-`betterlockscreen {{[-l|--lock]}} pixel --text "{{custom lock screen text}}"`
+`betterlockscreen {{[-l|--lock]}} pixel {{[-t|--text]}} "{{custom lock screen text}}"`
```

Comment:
Where did you find this info? None of the documentation mentions it and testing it on my own computer the program errors when using `-t`

Reply:
The english page was using `--text` while the translations were using `-t`. I figured I would combine them. Admittedly I did not test this out. So if the correct option is `--text` I can fix this change.